File: code/8f218.pas
program Bai218;
var a:array[1..1000] of integer;
    i, n, dem: integer;
begin
  readln(n);
  for i := 1 to n do read(a[i]);
  dem := 0;
  for i := 1 to n do 
        if a[i] mod 40 = 0 then dem := dem +1;
        write(dem);
end.

File: code/8f218.cpp
#include <iostream>
using namespace std;

int main() 
{
    int n,dem,i;
    int a[100000];
    cin>>n;
    for (i=1; i<=n; i++)
        cin>>a[i];
    dem=0;
    for (i=1; i<=n; i++)
        if (a[i]%40==0)
            dem++;
    cout<<dem;
    return 0;
}